docs/meshstack.openshift.index.md

```diff
@@ -218,8 +218,65 @@ roleRef:
   name: meshfed-service
 ```
 
+### Custom meshProject Roles
```

Comment:
i think we should also add it to the kubernetes docs

Reply:
Good point.